sdks/wasm/framework/src/WebAssembly.Net.WebSockets/ClientWebSocket.cs

```diff
@@ -540,106 +531,107 @@ public async Task<T> DequeuePayloadAsync (CancellationToken cancellationToken =
 			}
 		}
 
-		public sealed class ClientWebSocketOptions {
+		public sealed class ClientWebSocketOptions
+		{
 			private bool isReadOnly; // After ConnectAsync is called the options cannot be modified.
 			private readonly IList<string> requestedSubProtocols;
 
-			internal ClientWebSocketOptions ()
+			internal ClientWebSocketOptions()
```

Comment:
A bunch of formatting damage happened in this file, it should be reverted and if the file needs a formatting cleanup let's do that in a separate PR so the git blame is less confusing

Reply:
Addressed in recent commits